core/state_processor_rip7560.go

```diff
@@ -366,14 +388,19 @@ func ApplyRip7560ExecutionPhase(config *params.ChainConfig, vpr *ValidationPhase
 	return receipt, err
 }
 
-func prepareNonceValidationMessage(baseTx *types.Transaction, chainConfig *params.ChainConfig) *Message {
+func prepareNonceValidationMessage(baseTx *types.Transaction, chainConfig *params.ChainConfig) (*Message, bool) {
```

Comment:
The two return values seem to be exclusive, why not reduce them to one and differentiate them by whether they are nil or not?

Reply:
Yeah, that would be better. I'll fix.